ISSUE_TEMPLATE

```diff
@@ -0,0 +1,40 @@
+Before filing an issue:
+-----------------------
+- Attempt to reproduce the issue using 
+  [this sample application](https://github.com/forcedotcom/ServiceSDK-Android/)
+- Search existing issues, including issues that are closed.
+- Review the "Known Issues" 
+  [section of our release documentation](https://github.com/forcedotcom/ServiceSDK-Android/releases) 
+  for your issue.
+- If this is an implementation specific question please refer to the
+  [Snap-Ins for Android Developer Guide](https://developer.salesforce.com/docs/atlas.en-us.noversion.service_sdk_android.meta/service_sdk_android/servicesdk_android_dev_guide.htm)
+  or the [Service Cloud Snap-Ins API Reference](http://forcedotcom.github.io/ServiceSDK-Android/)
```

Comment:
Minor typo... add period to the end of the sentence.

Reply:
👍